### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by mgrachev**
#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed

### CODE REVIEW

**Comment by mgrachev** [comment]
@zanhsieh Is everything okay?

**Comment by zanhsieh** [comment]
@mgrachev 
Thanks for your contribution. I am not a helm member so the change on meta (means outside of each chart) should go thru helm members to merge it. I suggest you go to kubernetes slack #charts channel to shout out about this PR.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -105,9 +105,9 @@ This repository is used by Chart developers for maintaining the official charts
 * Provide a secure default configuration
 * Do not leverage alpha features of Kubernetes
 * Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
-* Follows [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices)
-  (especially for [labels](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)
-  and [values](https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md))
+* Follows [best practices](https://helm.sh/docs/chart_best_practices)
+  (especially for [labels](https://helm.sh/docs/chart_best_practices/#labels-and-annotations)
```
Comment (scottrigby) [inline CONTRIBUTING.md]:
```suggestion
  (especially for [labels](https://helm.sh/docs/topics/chart_best_practices/labels/)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -105,9 +105,9 @@ This repository is used by Chart developers for maintaining the official charts
...
+  and [values](https://helm.sh/docs/chart_best_practices/#values))
```
Comment (scottrigby) [inline CONTRIBUTING.md]:
```suggestion
  and [values](https://helm.sh/docs/topics/chart_best_practices/values/))
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -105,9 +105,9 @@ This repository is used by Chart developers for maintaining the official charts
 * Provide a secure default configuration
 * Do not leverage alpha features of Kubernetes
 * Includes a [NOTES.txt](https://github.com/helm/helm/blob/master/docs/charts.md#chart-license-readme-and-notes) explaining how to use the application after install
-* Follows [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices)
-  (especially for [labels](https://github.com/helm/helm/blob/master/docs/chart_best_practices/labels.md)
-  and [values](https://github.com/helm/helm/blob/master/docs/chart_best_practices/values.md))
+* Follows [best practices](https://helm.sh/docs/chart_best_practices)
```
Comment (scottrigby) [inline CONTRIBUTING.md]:
```suggestion
* Follows [best practices](https://helm.sh/docs/topics/chart_best_practices/)
```

File: REVIEW_GUIDELINES.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Chart Review Guidelines
 
-Anyone is welcome to review pull requests. Besides our [technical requirements](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements) and [best practices](https://github.com/helm/helm/tree/master/docs/chart_best_practices), here's an overview of process and review guidelines.
+Anyone is welcome to review pull requests. Besides our [technical requirements](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements) and [best practices](https://helm.sh/docs/chart_best_practices), here's an overview of process and review guidelines.
```
Comment (scottrigby) [inline REVIEW_GUIDELINES.md]:
```suggestion
Anyone is welcome to review pull requests. Besides our [technical requirements](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements) and [best practices](https://helm.sh/docs/topics/chart_best_practices/), here's an overview of process and review guidelines.
```

**Review by scottrigby — CHANGES_REQUESTED** [review]
Links have updated

**Comment by mgrachev** [comment]
@scottrigby Updated.

**Comment by mgrachev** [comment]
@scottrigby @mattfarina Is everything okay?

**Comment by mgrachev** [comment]
Ping

**Pull request closed without merging**